app/src/main/java/com/example/fooding/fragments/PantryFragment.java

```diff
@@ -51,6 +62,8 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
         recipeSearchView = view.findViewById(R.id.ingredientSearchView);
         ingredientsRecyclerView = view.findViewById(R.id.ingredientsRecyclerView);
         suggestedRecyclerView = view.findViewById(R.id.suggestedIngredientsRecyclerView);
+        ingredientName = view.findViewById(R.id.ingredientName);
+        ingredientImage = view.findViewById(R.id.ingredientImage); 
```

Comment:
These TextViews are created but never used `PantryFragment`. Maybe this should be removed from this PR and put into a separate PR when the TextViews are used. 

It's a good habit to keep PRs concise and free of unused code.

Reply:
okay, I have removed those lines of code.